charts/aws-cloud-controller-manager/templates/cluserrolebinding.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Values.clusterRoleName }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
how about "helm.sh/chart" instead of "chart", the helm docs recommend it https://helm.sh/docs/chart_best_practices/labels/

> helm.sh/chart | REC | This should be the chart name and version: {{ .Chart.Name }}-{{ .Chart.Version \\| replace "+" "_" }}.